src/middlewares/permissions.js

```diff
@@ -17,9 +19,10 @@ function createCheckPermission(environmentSecret, collectionName) {
 
   function checkPermissionListAndSearch(request, response, next) {
     const { searchToEdit } = request.query;
+    const renderingId = request.user.relationships.renderings.data[0].id;
```

Comment:
Maybe you could create a centralized function to extract the `renderingId` from the user context.
Twice the same line of code in this file.